test/e2e/tests/metrics_test/main_test.go

```diff
@@ -0,0 +1,46 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"os"
+	"testing"
+
+	eraserv1alpha1 "github.com/Azure/eraser/api/v1alpha1"
+	"github.com/Azure/eraser/test/e2e/util"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/e2e-framework/pkg/env"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/envfuncs"
+)
+
+func TestMain(m *testing.M) {
+	utilruntime.Must(eraserv1alpha1.AddToScheme(scheme.Scheme))
+
+	eraserImage := util.ParsedImages.EraserImage
+	managerImage := util.ParsedImages.ManagerImage
+
+	util.Testenv = env.NewWithConfig(envconf.New())
+	// Create KinD Cluster
+	util.Testenv.Setup(
+		envfuncs.CreateKindClusterWithConfig(util.KindClusterName, util.NodeVersion, "../../kind-config.yaml"),
+		envfuncs.CreateNamespace(util.TestNamespace),
+		util.DeployOtelCollector(util.TestNamespace),
+		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.ManagerImage),
+		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.Image),
+		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.VulnerableImage),
+		util.DeployEraserHelm(util.TestNamespace,
+			"--set", util.CollectorImageRepo.Set(""),
+			"--set", util.ScannerImageRepo.Set(""),
```

Comment:
Since the metrics work in the scanner, we should probably test that they work. I think we'll need to enable the collector and scanner, and check that their metrics are correct.

Reply:
when testing the scanner metrics with the pipeline, it can find other images that are vulnerable as well so theres no way to ensure that only one image will be found vulnerable and removed to check the metrics

Reply:
We will have to think of a way to test using the collector/scanner. If there's no way to test the eraser metrics with the collector/scanner, then we'll need another e2e test. That test should verify that the metrics are properly produced when scanning is enabled.